# Belarusian translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2012-06-14 00:53+0100\n"
"PO-Revision-Date: 2012-04-13 00:57+0000\n"
"Last-Translator: Soltan Dzmitry <Unknown>\n"
"Language-Team: Belarusian <be@li.org>\n"
"Language: be\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Launchpad-Export-Date: 2012-04-17 13:40+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:433 ../UpdateManager/Core/utils.py:433
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f КБ"
msgstr[1] "%(size).0f КБ"
msgstr[2] "%(size).0f КБ"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:436 ../UpdateManager/Core/utils.py:436
#, python-format
msgid "%.1f MB"
msgstr "%.1f МБ"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:436
#, python-format
msgid "Server for %s"
msgstr "Сервер для %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Галоўны сервер"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Адмысловыя серверы"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Немагчыма разлічыць запіс sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Немагчыма адшукаць ніводнага файла з пакетам, верагодна, гэты дыск не "
"з'яўляецца Ubuntu ці мае іншую архітэктуру."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Немагчыма дадаць дыск"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Пры дадаванні кампакт-дыска адбылася памылка, абнаўленне будзе спынена. Калі "
"ласка, паведаміце аб памылцы, калі гэта быў карэктны кампакт-дыск Ubuntu.\n"
"\n"
"Тэкст памылкі:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:151
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Выдаліць пакет, пазначаны як памылковы"
msgstr[1] "Выдаліць пакеты, пазначаныя як памылковыя"
msgstr[2] "Выдаліць пакеты ў кепскім стане"

#: ../DistUpgrade/DistUpgradeCache.py:154
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Пакет “%s“ знаходзіцца ў супярэчлівым стане і мусіць быць пераўсталяваны, "
"але немагчыма адшукаць архіў для яго. Ці жадаеце выдаліць зараз гэты пакет?"
msgstr[1] ""
"Пакеты “%s“ знаходзяцца ў супярэчлівым стане і мусіць быць пераўсталяваны, "
"але немагчыма адшукаць архівы для іх. Ці жадаеце выдаліць зараз гэтыя пакеты?"
msgstr[2] ""
"Пакеты “%s“ знаходзяцца ў супярэчлівым стане і мусіць быць пераўсталяваны, "
"але немагчыма адшукаць архівы для іх. Ці жадаеце выдаліць зараз гэтыя пакеты?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:255
msgid "The server may be overloaded"
msgstr "Магчыма сервер перагружаны"

#: ../DistUpgrade/DistUpgradeCache.py:368
msgid "Broken packages"
msgstr "Пакеты з памылкамі"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Сістэма змяшчае пашкоджаныя пакеты, якія не могуць быць выпраўленыя гэтай "
"праграмай.  Калі ласка, спачатку выпраўце іх з дапамогай synaptic ці apt-get "
"перш чым працягваць."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Невырашальная праблема паўстала падчас падліку апгрэйду:\n"
"%s\n"
"\n"
" Гэта магло здарыцца з-за наступных прычын:\n"
" * апгрэйд да прэ-рэлізу Ubuntu\n"
" * працы з прэ-рэлізам Ubuntu\n"
" * неафіцыйных праграм, якія не пастаўляюцца з Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:703
msgid "This is most likely a transient problem, please try again later."
msgstr "Магчыма, гэта часовая праблема. Паспрабуйце паўтарыць пазней."

#: ../DistUpgrade/DistUpgradeCache.py:706
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Калі з прапанаванага нічога не пасуе, калі ласка, адпраўце гэту справаздачу, "
"выкарыстоўваючы ў тэрмінале каманду 'ubuntu-bug update-manager'"

#: ../DistUpgrade/DistUpgradeCache.py:711
#: ../UpdateManager/UpdateManager.py:1031
msgid "Could not calculate the upgrade"
msgstr "Немагчыма падлічыць абнаўленне"

#: ../DistUpgrade/DistUpgradeCache.py:762
msgid "Error authenticating some packages"
msgstr "Памылка аўтэнтыфікацыі некаторых пакетаў"

#: ../DistUpgrade/DistUpgradeCache.py:763
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Немагчыма аўтэнтыфікаваць некаторыя пакеты. Гэта можа быць з-за часовай "
"праблемы з сеткай. Вы можаце паспрабаваць паўтарыць дзеяньне пазней. "
"Глядзіце ніжэй спіс неаўтэнтыфікаваных пакетаў."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "Пакет “%s“ пазначаны для выдалення, але ён у чорным спісе выдалення."

#: ../DistUpgrade/DistUpgradeCache.py:787
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Важны пакет “%s“ пазначаны для выдалення."

#: ../DistUpgrade/DistUpgradeCache.py:796
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Спроба ўсталёўкі ўнесенай у чорны спіс версіі '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:914
#, python-format
msgid "Can't install '%s'"
msgstr "Немагчыма ўсталяваць \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Немагчыма ўсталяваць патрэбны пакет. Калі ласка, адпраўце гэтую справаздачу, "
"выкарыстоўваючы ў тэрмінале каманду 'ubuntu-bug update-manager'"

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:926
msgid "Can't guess meta-package"
msgstr "Немагчыма вызначыць мета-пакет"

#: ../DistUpgrade/DistUpgradeCache.py:927
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Вашая сістэма не ўтрымлівае пакет ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop ці edubuntu-desktop таму немагчыма вызначыць якую версію Ubuntu вы "
"маеце.\n"
"Калі ласка, найперш усталюйце адзін з вышэй узгаданых пакетаў з дапамогай "
"synaptic ці apt-get."

#: ../DistUpgrade/DistUpgradeController.py:114
msgid "Reading cache"
msgstr "Чытанне кэша"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid "Unable to get exclusive lock"
msgstr "Памылка пры праверцы аўтэнтычнасці некаторых пакетаў"

#: ../DistUpgrade/DistUpgradeController.py:224
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Звычайна, гэта азначае, што іншы кіраўнік пакетаў (напрыклад apt-get ці "
"aptitude) ужо працуе. Калі ласка, закройце іншае дастасаванне."

#: ../DistUpgrade/DistUpgradeController.py:257
msgid "Upgrading over remote connection not supported"
msgstr "Апгрэйд праз аддаленае падлучэнне не падтрымліваецца"

#: ../DistUpgrade/DistUpgradeController.py:258
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Вы спрабуеце выканаць абнаўленьне праз ssh-злучэнне з непадтрымліваемым "
"кліентам. Абнавіцеся ў тэкставым рэжыме праз \"do-release-upgrade\".\n"
"\n"
"Абнаўленне прыпынена. Паспрабуйце без ssh."

#: ../DistUpgrade/DistUpgradeController.py:272
msgid "Continue running under SSH?"
msgstr "Працягваць выкананне праз SSH?"

#: ../DistUpgrade/DistUpgradeController.py:273
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Гэты сеанс запушчаны праз ssh. Не рэкамендавана выконваць абнаўленьне праз "
"ssh, бо ў выпадку няўдачы аднаўленне будзе вельмі складаным.\n"
"\n"
"Калі Вы працягнеце, дадатковая служба ssh будзе запушчана на порце «%s».\n"
"Жадаеце працягнуць?"

#: ../DistUpgrade/DistUpgradeController.py:287
msgid "Starting additional sshd"
msgstr "Запуск дадатковай sshd"

#: ../DistUpgrade/DistUpgradeController.py:288
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Каб зрабіць аднаўленне ў выпадку памылкі лягчэйшым, на порце '%s' будзе "
"запушчаная дадатковая служба sshd. Калі нешта здарыцца, з дапамогай ssh Вы "
"зможаце далучыцца да дадзенай службы.\n"

#: ../DistUpgrade/DistUpgradeController.py:296
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Калі ў вас уключаны брандмаўэр, вам магчыма трэба часова адкрыць гэты порт. "
"Паколькі гэта патэнцыйна небяспечна, порт не адкрыецца аўтаматычна. Вы "
"можаце адкрыць наступны спосабам:\n"
"«%s» порт."

#: ../DistUpgrade/DistUpgradeController.py:368
#: ../DistUpgrade/DistUpgradeController.py:413
msgid "Can not upgrade"
msgstr "Немагчыма абнавіць"

#: ../DistUpgrade/DistUpgradeController.py:369
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Абнаўленне ад '%s' да '%s' не падтрымліваецца дадзенай прыладай."

#: ../DistUpgrade/DistUpgradeController.py:378
msgid "Sandbox setup failed"
msgstr "Усталёўка бяспечнага асяроддзя не атрымалася"

#: ../DistUpgrade/DistUpgradeController.py:379
msgid "It was not possible to create the sandbox environment."
msgstr "Не атрымалася стварыць бяспечнае асяроддзе."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid "Sandbox mode"
msgstr "Рэжым бяспечнага асяроддзя («пясочніцы»)"

#: ../DistUpgrade/DistUpgradeController.py:386
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Гэта абнаўленне запушчана ў бяспечным (тэставым) асяроддзі. Усе змены "
"запісваюцца ў '%s'  і будуць страчаныя пасля перазагрузкі.\n"
"\n"
"Да наступнай перазагрузкі ніякіх зменаў у сістэмным каталогу праводзіцца не "
"будзе."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python усталяваны некарэктна. Калі ласка, выпраўце сімвалічную спасылку \"/"
"usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:440
msgid "Package 'debsig-verify' is installed"
msgstr "Пакет 'debsig-verify' усталяваны"

#: ../DistUpgrade/DistUpgradeController.py:441
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Абнаўленне ня можа працягвацца, калі ўсталаваны гэты пакет.\n"
"Спачатку выдаліце яго ў Synaptic або з дапамогай 'apt-get remove debsig-"
"verify' і запуьціце абнаўленне ізноў."

#: ../DistUpgrade/DistUpgradeController.py:453
#, python-format
msgid "Can not write to '%s'"
msgstr "Немагчыма запісаць у '%s'"

#: ../DistUpgrade/DistUpgradeController.py:454
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Няма доступу да сістэмнага каталога '%s'  у вашай сістэме. Абнаўленне не "
"можа працягвацца.\n"
" Калі ласка, пераканайцеся ў наяўнасці доступу да сістэмнага каталога."

#: ../DistUpgrade/DistUpgradeController.py:465
msgid "Include latest updates from the Internet?"
msgstr "Загрузіць апошнія абнаўленні з інтэрнэту?"

#: ../DistUpgrade/DistUpgradeController.py:466
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Апошнія абнаўленні могуць быць загружаны з Інтэрнэту і ўсталяваны пры "
"абнаўленні ўсёй сістэмы. Калі вы падключаны да Інтэрнэту, гэта настойліва "
"рэкамендуецца зрабіць.\n"
"\n"
"Абнаўленне сістэмы — гэта доўгі працэс, але па завяршэнні, ваша сістэма "
"будзе ў актуальным стане. Вы можаце гэтага не рабіць, але рэкамендуем "
"усталяваць апошнія абнаўленні як мага хутчэй.\n"
"Калі вы адкажаце «Не», то абнаўленні праз Інтэрнет загружаны ня будуць."

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid "disabled on upgrade to %s"
msgstr "заблакавана пры абнаўленні да %s"

#: ../DistUpgrade/DistUpgradeController.py:713
msgid "No valid mirror found"
msgstr "Не знойдзена ніводнага дзейснага люстэрка"

#: ../DistUpgrade/DistUpgradeController.py:714
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"У працэсе сканавання інфармацыі аб Вашых сховішчах, не было знойдзена "
"люстэрка для абнаўлення дыстрыбутыву. Такое магчыма ў выпадку, калі Вы "
"выкарыстоўваеце ўнутранае люстэрка, альбо інфармацыя аб люстэрках "
"састарэла.\n"
"\n"
"Ці жадаеце Вы перазапісаць файл 'sources.list'? Калі Вы абярэце «Так», то "
"будуць абноўлены ўсе '%s' запісаў да '%s'.\n"
"Калі ж Вы абярэце «Не», то абнаўленне будзе адменена."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:734
msgid "Generate default sources?"
msgstr "Згенераваць крыніцы па змаўчанні?"

#: ../DistUpgrade/DistUpgradeController.py:735
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Ня знойдзены запіс для '%s' у файле 'sources.list'.\n"
"\n"
"Дадаць стандартны запіс для '%s'? Выбар «Не» азначае адмову ад абнаўлення."

#: ../DistUpgrade/DistUpgradeController.py:770
msgid "Repository information invalid"
msgstr "Інфармацыя аб сховішчы няслушная"

#: ../DistUpgrade/DistUpgradeController.py:771
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:778
msgid "Third party sources disabled"
msgstr "Зыходнікі ад трэціх бакоў - адключаныя"

#: ../DistUpgrade/DistUpgradeController.py:779
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Некаторыя пабочныя крыніцы ў файле «sources.list» былі адключаныя. Вы "
"зможаце іх зноў уключыць пасля абнаўлення з дапамогай утыліты «Крыніцы "
"ўсталёўкі» альбо вашага мэнэджара пакункаў."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакунак у нестабільным стане"
msgstr[1] "Пакункі ў няўстойлівым стане"
msgstr[2] "Пакункаў у наўстойлівым стане"

#: ../DistUpgrade/DistUpgradeController.py:822
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Пакунак '%s' няўстойлівы і павінен быць пераўсталяваны, аднак, для яго ня "
"знойдзен адпаведны архіў. Калі ласка, пераўсталюйце гэты пакунак уручную "
"альбо выдаліце яго з сістэмы."
msgstr[1] ""
"Пакункі '%s' няўстойлівыя і павінны быць пераўсталяваны, аднак, для іх ня "
"знойдзены адпаведныя архівы. Калі ласка, пераўсталюйце гэтыя пакункі уручную "
"альбо выдаліце іх з сістэмы."
msgstr[2] ""
"Пакункі '%s' няўстойлівыя і павінны быць пераўсталяваны, аднак, для іх ня "
"знойдзены адпаведныя архівы. Калі ласка, пераўсталюйце гэтыя пакункі уручную "
"альбо выдаліце іх з сістэмы."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Error during update"
msgstr "Памылка падчас абнаўлення"

#: ../DistUpgrade/DistUpgradeController.py:871
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Пры абнаўленні паўстала праблема. Звычайна гэта бывае выклікана праблемамі ў "
"сетцы. Праверце сеткавыя злучэнні і паспрабуйце яшчэ."

#. print("on_button_install_clicked")
#: ../DistUpgrade/DistUpgradeController.py:880
#: ../UpdateManager/UpdateManager.py:757
msgid "Not enough free disk space"
msgstr "Бракуе дыскавае прасторы"

#: ../DistUpgrade/DistUpgradeController.py:881
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Абнаўленне было спынена. Для абнаўлення патрабуецца агулам %s вольнай "
"прасторы на дыску '%s'. Калі ласка, вызваліце ня меньш чым %s дадатковага "
"месца на дыску '%s'. Ачысціце сметніцу і выдаліце часовыя пакеты былых "
"усталёвак з дапамогай 'sudo apt-get clean'."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:910
#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Calculating the changes"
msgstr "Падлічыць змены"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:942
msgid "Do you want to start the upgrade?"
msgstr "Ці жадаеце пачаць абнаўленне?"

#: ../DistUpgrade/DistUpgradeController.py:1008
msgid "Upgrade canceled"
msgstr "Абнаўленне скасавана"

#: ../DistUpgrade/DistUpgradeController.py:1009
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Гэта абнаўленне будзе зараз скасавана і адбудзецца аднаўленне зыходнага "
"стану сістэмы. Вы можаце працягнуць гэта абнаўленне пазней."

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1149
msgid "Could not download the upgrades"
msgstr "Не атрымалася загрузіць абнаўленні"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Абнаўленне было перапынена. Калі ласка, праверце ваша злучэнне з інтэрнэтам "
"або крыніцу ўсталёўкі і паспрабуйце зноў. Усе загружаныя файлы былі "
"захаваныя."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1100
#: ../DistUpgrade/DistUpgradeController.py:1137
#: ../DistUpgrade/DistUpgradeController.py:1242
msgid "Error during commit"
msgstr "Памылка пры фіксаванні"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1102
#: ../DistUpgrade/DistUpgradeController.py:1139
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid "Restoring original system state"
msgstr "Аднаўленне першапачатковага стану сістэмы"

#: ../DistUpgrade/DistUpgradeController.py:1103
#: ../DistUpgrade/DistUpgradeController.py:1118
#: ../DistUpgrade/DistUpgradeController.py:1140
msgid "Could not install the upgrades"
msgstr "Немагчыма ўсталяваць абнаўленні"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Абнаўленне было скасавана. Ваша сістэма можа апынуцца ў стане непрыдатным "
"для нармальнага выкарыстання. Зараз будзе запушчаны працэс аднаўлення (dpkg "
"--configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1113
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/update-manager/+filebug and attach the files in /var/log/dist-"
"upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1150
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Абнаўленне спыненае. Калі ласка праверце злучэнне з Інтэрнэтам, альбо іншую "
"крыніцу ўсталёўкі і паспрабуйце зноў. "

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid "Remove obsolete packages?"
msgstr "Выдаліць састарэлыя пакеты?"

#: ../DistUpgrade/DistUpgradeController.py:1231
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "За_хаваць"

#: ../DistUpgrade/DistUpgradeController.py:1231
msgid "_Remove"
msgstr "_Выдаліць"

#: ../DistUpgrade/DistUpgradeController.py:1243
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "Падчас ачысткі паўстала праблема. Падрабязнасці выкладзены ніжэй. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1319
msgid "Required depends is not installed"
msgstr "Неабходныя залежнасці не ўсталяваныя"

#: ../DistUpgrade/DistUpgradeController.py:1320
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Неабходныя залежнасці \"%s\" не ўсталяваныя. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1588
#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Checking package manager"
msgstr "Праверка мэнэджара пакетаў"

#: ../DistUpgrade/DistUpgradeController.py:1593
msgid "Preparing the upgrade failed"
msgstr "Збой падрыхтоўкі да абнаўлення"

#: ../DistUpgrade/DistUpgradeController.py:1594
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Getting upgrade prerequisites failed"
msgstr "Падрыхтоўка да абнаўлення завяршылася няўдала"

#: ../DistUpgrade/DistUpgradeController.py:1609
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "Updating repository information"
msgstr "Абнаўленне інфармацыі аб сховішчы"

#: ../DistUpgrade/DistUpgradeController.py:1644
msgid "Failed to add the cdrom"
msgstr "Дадаванне кампакт-дыска завяршылася няўдала"

#: ../DistUpgrade/DistUpgradeController.py:1645
msgid "Sorry, adding the cdrom was not successful."
msgstr "Выбачайце, даданне кампакт-дыска завяршылася няўдала."

#: ../DistUpgrade/DistUpgradeController.py:1673
msgid "Invalid package information"
msgstr "Некарэктная інфармацыя аб пакеце"

#: ../DistUpgrade/DistUpgradeController.py:1674
msgid "After updating your package "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1698
#: ../DistUpgrade/DistUpgradeController.py:1750
msgid "Fetching"
msgstr "Атрыманне"

#: ../DistUpgrade/DistUpgradeController.py:1704
#: ../DistUpgrade/DistUpgradeController.py:1754
msgid "Upgrading"
msgstr "Абнаўленне"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1709
#: ../DistUpgrade/DistUpgradeController.py:1756
#: ../DistUpgrade/DistUpgradeController.py:1763
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Upgrade complete"
msgstr "Абнаўленне скончана"

#: ../DistUpgrade/DistUpgradeController.py:1710
#: ../DistUpgrade/DistUpgradeController.py:1757
#: ../DistUpgrade/DistUpgradeController.py:1764
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Абнаўленне скончана, аднак падчас працэсу абнаўлення здарыліся памылкі."

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Searching for obsolete software"
msgstr "Пошук састарэлых праграм"

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "System upgrade is complete."
msgstr "Абнаўленне сістэмы завершана"

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid "The partial upgrade was completed."
msgstr "Частковае абнаўленне завершана."

#: ../DistUpgrade/DistUpgradeQuirks.py:204
msgid "evms in use"
msgstr "evms ужываецца"

#: ../DistUpgrade/DistUpgradeQuirks.py:205
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Ваша сістэма выкарыстоўвае мэнэджар тамоў 'evms' в /proc/mounts. Праргама "
"'evms' болей не падтрымліваецца.  Калі ласка, зачыніце яе і запусціце "
"абнаўленне зноў."

#: ../DistUpgrade/DistUpgradeQuirks.py:502
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:504
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:526
#: ../DistUpgrade/DistUpgradeQuirks.py:554
#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Абнаўленне можа выклікаць зніжэнне якасці эфектаў працоўнага стала і "
"прадукцыйнасці ў гульнях і праграмах, што актыўна выкарыстоўваюць графіку."

#: ../DistUpgrade/DistUpgradeQuirks.py:530
#: ../DistUpgrade/DistUpgradeQuirks.py:558
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"На гэтым кампутары выкарыстоўваюцца драйвера NVIDIA. Няма даступных версій "
"гэтых драйвераў, якія працавалі бы з Вашай відэакарткай у Ubuntu 10.04 LTS\n"
"\n"
"Працягнуць?"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"На гэтым кампутары выкарыстоўваюцца драйвера AMD 'fglrx'. Няма даступных "
"версій гэтых драйвераў, якія працавалі бы з Вашым абсталяваннем ва 10.04 "
"LTS\n"
"\n"
"Працягнуць?"

#: ../DistUpgrade/DistUpgradeQuirks.py:615
msgid "No i686 CPU"
msgstr "Няма i686-сумяшчальнага працэсара"

#: ../DistUpgrade/DistUpgradeQuirks.py:616
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ваша сістэма выкарыстоўвае i568-сумяшчальны працэсар, альбо працэсар у якім "
"няма пашырэння «cmov». Усе пакеты былі сабраны з аптымізацыяй пад "
"архітэктуру i686 і вышэй. Абнавіць вашу сістэму да новай версіі Ubuntu на "
"гэтым кампутары не атрымаецца."

#: ../DistUpgrade/DistUpgradeQuirks.py:652
msgid "No ARMv6 CPU"
msgstr "Няма працэсара ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:653
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ваша сістэма выкарыстоўвае працэсар архітэктуры ARM, старэйшы за архітэктуру "
"ARMv6. Усе пакункі ў karmic былі пабудаваны з аптымізацыяй пад архітэктуру "
"ARMv6 і вышэй. Вашу сістэму немагчыма абнавіць да новага рэлізу Ubuntu з "
"бягучым апаратным забеспячэннем."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "No init available"
msgstr "Служба init недаступна"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Падобна на тое, што Ваша сістэма з'яўляецца віртуалізаваным асяроддзем без "
"службы init, напрыклад Linux-VServer.\n"
"\n"
"Вы ўпэўнены, што хочаце працягнуць?"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Sandbox upgrade using aufs"
msgstr "Абнаўленне ў бяспечным асяроддзі з дапамогай aufs"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Выкарыстоўваць дадзены шлях для пошука кампакт-дыска з пакункамі абнаўленняў."

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Выкарстоўваць інтэрфэйс. Зараз даступны: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "*DEPRECATED* this option will be ignored"
msgstr "Гэты параметр САСТАРЭЎ і не будзе ўлічвацца"

#: ../DistUpgrade/DistUpgradeMain.py:79
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Падрыхтаваць толькі частковае абнаўленне (sources.list ня будзе перазапісаны)"

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Disable GNU screen support"
msgstr "Адключыць падтрымку экрана GNU"

#: ../DistUpgrade/DistUpgradeMain.py:84
msgid "Set datadir"
msgstr "Наладзіць каталог з дадзенымі"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:195
#: ../UpdateManager/DistUpgradeFetcherKDE.py:155
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Калі ласка, устаўце \"%s\" у прыладу \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:209
msgid "Fetching is complete"
msgstr "Загрузка скончана"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:222
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Загрузка файла %li з %li на хуткасці %sБайт/с"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:296
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:309
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:371
#, python-format
msgid "About %s remaining"
msgstr "Засталося прыблізна %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:225
#, python-format
msgid "Fetching file %li of %li"
msgstr "Загрузка файла %li з %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
msgid "Applying changes"
msgstr "Ужыванне зменаў"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:212
#: ../DistUpgrade/DistUpgradeViewKDE.py:275
msgid "dependency problems - leaving unconfigured"
msgstr "праблемы залежнасьцяў — пакідаем неналаджанымі"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:277
#, python-format
msgid "Could not install '%s'"
msgstr "Немагчыма ўсталяваць \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:278
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Абнаўленне будзе працягнута, але '%s' пакунак(аў) могуць быць у непрацоўным "
"стане. Калі ласка, разглядзіце магчымасць аб паданні справаздачы на конт "
"памылкі."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Замяніць адмысловы файл наладкі\n"
" \"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Вы страціце ўсе змены, якія зрабілі ў гэтым файле канфігурацыі, калі "
"заменіце яго новай версіяй."

#: ../DistUpgrade/DistUpgradeViewGtk.py:251
#: ../DistUpgrade/DistUpgradeViewGtk3.py:256
#: ../DistUpgrade/DistUpgradeViewKDE.py:323
msgid "The 'diff' command was not found"
msgstr "Ня знойдзена праграма \"diff\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:477
#: ../DistUpgrade/DistUpgradeViewText.py:92
msgid "A fatal error occurred"
msgstr "Адбылася крытычная памылка"

#: ../DistUpgrade/DistUpgradeViewGtk.py:465
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Калі ласка, паведаміце аб гэтым як аб памылцы (калі вы яшчэ гэтага не "
"зрабілі) і далучыце файлы /var/log/dist-upgrade/main.log и /var/log/dist-"
"upgrade/apt.log у вашу справаздачу. Абнаўленне было скасавана.\n"
"Ваш арыгінальны файл sources.list быў захаваны ў  /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:495
msgid "Ctrl-c pressed"
msgstr "націснута Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:483
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Гэта скасуе працэс і сістэма можа стаць непрацаздольнай. \r\n"
"Вы сапраўды хочаце зрабіць гэта ?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:631
#: ../DistUpgrade/DistUpgradeViewGtk3.py:629
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Каб пазбегнуць страты дадзеных, зачыніце ўсе адчыненыя праграмы і дакументы."

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:643
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Больш не падтрымліваецца Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Усталяванне старой версіі (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "Remove (%s)"
msgstr "Выдаліць (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "No longer needed (%s)"
msgstr "Больш не патрэбны (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "Install (%s)"
msgstr "Усталяваць (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Upgrade (%s)"
msgstr "Абнавіць (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#: ../UpdateManager/DistUpgradeFetcherKDE.py:157
msgid "Media Change"
msgstr "Змена носьбіта"

#: ../DistUpgrade/DistUpgradeViewKDE.py:335
msgid "Show Difference >>>"
msgstr "Паказаць адрозненні >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:338
msgid "<<< Hide Difference"
msgstr "<<< Схаваць адрозненні"

#: ../DistUpgrade/DistUpgradeViewKDE.py:554
msgid "Error"
msgstr "Памылка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:568
msgid "&Cancel"
msgstr "&Скасаваць"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "&Close"
msgstr "&Зачыніць"

#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "Show Terminal >>>"
msgstr "Паказаць тэрмінал >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:621
msgid "<<< Hide Terminal"
msgstr "<<< Схаваць тэрмінал"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Information"
msgstr "Інфармацыя"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
#: ../DistUpgrade/DistUpgradeViewKDE.py:796
#: ../DistUpgrade/DistUpgradeViewKDE.py:799 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Дэталі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:777
#, python-format
msgid "No longer supported %s"
msgstr "Больш не падтрымліваецца (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "Remove %s"
msgstr "Выдаліць %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgradeViewText.py:182
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Выдаліць %s (было ўсталявана аўтаматычна)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Install %s"
msgstr "Усталёўка %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Upgrade %s"
msgstr "Абнаўленне %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
#: ../DistUpgrade/DistUpgradeViewText.py:230
msgid "Restart required"
msgstr "Патрабуецца перазагрузка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Перагрузіце сістэму для завяршэння абнаўлення</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:812 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "Пера_запусціць"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:830
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Перарваць абнаўленне?</big></b>\n"
"Калі вы перапыніце абнаўленне, сістэма можа працаваць нестабільна. "
"Настойліва рэкамендуем працягваць абнаўленне."

#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid "Cancel Upgrade?"
msgstr "Скасаваць абнаўленне?"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li дзень"
msgstr[1] "%li дні"
msgstr[2] "%li дзён"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li гадзіна"
msgstr[1] "%li гадзіны"
msgstr[2] "%li гадзін"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li хвіліна"
msgstr[1] "%li хвіліны"
msgstr[2] "%li хвілін"

#: ../DistUpgrade/DistUpgradeView.py:66
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li сэкунда"
msgstr[1] "%li сэкунды"
msgstr[2] "%li сэкундаў"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:100
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Загрузка працягнецца каля %s пры 1Мбіт DSL злучэнні і каля %s пры мадэмнам "
"злучэнні на хуткасці 56Кбіт."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:155
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Загрузка працягнецца прыкладна %s на Вашам далучэнні. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Падрыхтоўка да абнаўлення"

#: ../DistUpgrade/DistUpgradeView.py:260
msgid "Getting new software channels"
msgstr "Змена крыніц усталёўкі"

#: ../DistUpgrade/DistUpgradeView.py:261 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Атрымаць новыя пакеты"

#: ../DistUpgrade/DistUpgradeView.py:262 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Усталяваць абнаўленні"

#: ../DistUpgrade/DistUpgradeView.py:263 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Ачыстка"

#: ../DistUpgrade/DistUpgradeView.py:348
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d усталяваны пакет больш не падтрымліваецца Canonical. Вы можаце "
"атрымаць падтрымку ў суполцы."
msgstr[1] ""
"%(amount)d усталяваных пакета больш не падтрымліваюцца Canonical. Вы можаце "
"атрымаць падтрымку ў суполцы."
msgstr[2] ""
"%(amount)d усталяваных пакетаў больш не падтрымліваюцца Canonical. Вы можаце "
"атрымаць падтрымку ў суполцы."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d пакет будзе выдалены."
msgstr[1] "%d пакеты будзе выдалена."
msgstr[2] "%d пакетаў будзе выдалена."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d новы пакет буде ўсталяваны."
msgstr[1] "%d новых пакеты будзе ўсталявана."
msgstr[2] "%d новых пакетаў будзе ўсталявана."

#: ../DistUpgrade/DistUpgradeView.py:368
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d пакет будзе абноўлены."
msgstr[1] "%d пакеты будзе абноўлена."
msgstr[2] "%d пакетаў будзе абноўлена."

#: ../DistUpgrade/DistUpgradeView.py:373
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Усяго неабходна загрузіць %s. "

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:382
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:387
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:392 ../UpdateManager/UpdateManager.py:676
msgid "The software on this computer is up to date."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Для вашае сістэмы няма абнаўленняў. Абнаўленне будзе скасавана."

#: ../DistUpgrade/DistUpgradeView.py:406
msgid "Reboot required"
msgstr "Патрабуецца перазагрузка"

#: ../DistUpgrade/DistUpgradeView.py:407
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "Абнаўленне скончанае і патрабуецца перазагрузка. Перазагрузіцца зараз?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:72
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "аўтэнтыфікаваць '%(file)s' замест '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#, python-format
msgid "extracting '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:151
#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:151
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid "Could not run the upgrade tool"
msgstr "Не атрымалася запусціць сродак абнаўлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"У праграме абнаўлення, верагодна, памылка. Калі ласка, адпраўце справаздачу, "
"выкарыстоўваючы ў тэрмінале каманду 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Подпіс прылады абнаўлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Прылада абнаўлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Не магу атрымаць"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Не магу атрымаць абнаўленне. Магчыма, паўстала праблема з сецівам. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Няўдалая ідэнтыфікацыя"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Праверка ідэнтыфікацыі абнаўлення не атрымалася. Магчыма, паўстала праблема "
"з сецівам або на серверы. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Не магу выняць"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Не атрымалася выняць абнаўленне. Магчыма, паўстала праблема з сецівам або на "
"серверы. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Праверка не атрымалася"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Праверка абнаўлення завяршылася няўдала. Магчыма, паўстала праблема з "
"сецівам або на серверы. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:300
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Не атрымалася запусціць працэс абнаўлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Звычайна паўстае ў сістэме, дзе / tmp змантаваны пад noexec. Калі ласка, "
"перамантуйце без noexec і запусціце абнаўленне зноў."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Паведамленне аб памылцы «%s»."

#: ../DistUpgrade/DistUpgradeViewText.py:93
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Калі ласка, паведаміце пра гэта як пра памылку і ўключаюць у сябе файлы /var/"
"log/dist-upgrade/main.log і /var/log/dist-upgrade/apt.log ў Вашым дакладзе. "
"Абнаўленне не сфармавалася.\n"
"Імя арыгінальнае sources.list быў захаваны ў /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:117
msgid "Aborting"
msgstr "Абарваць"

#: ../DistUpgrade/DistUpgradeViewText.py:122
msgid "Demoted:\n"
msgstr "Паніжана:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:129
msgid "To continue please press [ENTER]"
msgstr "Каб працягнуць, націсніце [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [yN] "
msgstr "Працягнуць [тН] "

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Details [d]"
msgstr "Дэталі [д]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:165
#: ../DistUpgrade/DistUpgradeViewText.py:213
msgid "n"
msgstr "н"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "d"
msgstr "д"

#: ../DistUpgrade/DistUpgradeViewText.py:173
#, python-format
msgid "No longer supported: %s\n"
msgstr "Больш не падтрымліваецца: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, python-format
msgid "Remove: %s\n"
msgstr "Выдаліць: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:188
#, python-format
msgid "Install: %s\n"
msgstr "Усталяваць: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:193
#, python-format
msgid "Upgrade: %s\n"
msgstr "Абнавіць: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "Continue [Yn] "
msgstr "Працягнуць [Тн] "

#: ../DistUpgrade/DistUpgradeViewText.py:231
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Каб завяршыць абнаўленне, спатрэбіцца перазагрузка.\n"
"Калі вы абярэце «т», сістэма будзе перазагружана."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "Ска_саваць абнаўленне"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Пра_цягваць абнаўленне"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Скасаваць працэс абнаўлення?</big></b>\n"
"\n"
"Калі вы перапыніце абнаўленне, сістэма можа апынуцца ў непрацаздольным "
"стане. Настойліва рэкамендуецца працягваць абнаўленне."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "За_пусціць абнаўленне"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "За_мяніць"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Розьніца паміж файламі"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "Па_ведаміць аб памылцы"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Працягнуць"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Пачаць абнаўленне?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Перазагрузіце сістэму, каб завяршыць абнаўленне</big></b>\n"
"\n"
"Калі ласка, захавайце Вашу працу перад працягам."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Абнавіць дыстрыбутыў"

#: ../DistUpgrade/DistUpgrade.ui.h:19
#, fuzzy
msgid "<b><big>Upgrading Ubuntu to version 12.10</big></b>"
msgstr "<b><big>Абнаўленне Ubuntu да версіі 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Усталёўка новых крыніц праграмнага забеспячэння"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Перазапуск кампутара"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Тэрмінал"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:64
msgid "Please wait, this can take some time."
msgstr "Калі ласка, пачакайце. Гэта можа заняць пэўны час."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:66
msgid "Update is complete"
msgstr "Абнаўленне завершана"

#: ../UpdateManager/DistUpgradeFetcher.py:114
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Could not find the release notes"
msgstr "Не атрымалася знайсці заўвагі да выпуску"

#: ../UpdateManager/DistUpgradeFetcher.py:115
#: ../UpdateManager/DistUpgradeFetcherKDE.py:110
msgid "The server may be overloaded. "
msgstr "Магчыма, сервер перагружаны. "

#: ../UpdateManager/DistUpgradeFetcher.py:125
#: ../UpdateManager/DistUpgradeFetcherKDE.py:114
msgid "Could not download the release notes"
msgstr "Не атрымалася загрузіць заўвагі да выпуску"

#: ../UpdateManager/DistUpgradeFetcher.py:126
#: ../UpdateManager/DistUpgradeFetcherKDE.py:115
msgid "Please check your internet connection."
msgstr "Калі ласка, праверце вашае злучэнне з інтэрнэтам."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:68
#: ../UpdateManager/DistUpgradeFetcherKDE.py:91
msgid "Upgrade"
msgstr "Абнаўленне"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:95
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Нататкі да выпуску"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:134
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Downloading additional package files..."
msgstr "Загрузка дадатковых файлаў пакетаў ..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s з %s на хуткасці %sБ/с"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
#, python-format
msgid "File %s of %s"
msgstr "Файл %s з %s"

#: ../UpdateManager/ChangelogViewer.py:75
msgid "Open Link in Browser"
msgstr "Адкрыць спасылку ў браўзары"

#: ../UpdateManager/ChangelogViewer.py:78
msgid "Copy Link to Clipboard"
msgstr "Капіяваць спасылку ў буфер"

#: ../UpdateManager/GtkProgress.py:162
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Загрузка файла %(current)li з %(total)li з хуткасцю %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:167
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Загрузка файла %(current)li з %(total)li"

#: ../UpdateManager/UpdateManager.py:106 ../do-release-upgrade:100
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ваш выпуск Ubuntu больш не падтрымліваецца."

#: ../UpdateManager/UpdateManager.py:107
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""

#: ../UpdateManager/UpdateManager.py:115
msgid "Upgrade information"
msgstr "Інфармацыя аб абнаўленні"

#: ../UpdateManager/UpdateManager.py:233
#: ../UpdateManagerText/UpdateManagerText.py:35
msgid "Install"
msgstr "Усталяваць"

#: ../UpdateManager/UpdateManager.py:235
msgid "Name"
msgstr "Назва"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:395
#, python-format
msgid "Version %s: \n"
msgstr "Версія %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr "Не выяўлена сеткавае злучэнне, немагчыма загрузіць спіс змяненняў."

#: ../UpdateManager/UpdateManager.py:463
msgid "Downloading list of changes..."
msgstr "Загрузка спісу змяненняў ..."

#: ../UpdateManager/UpdateManager.py:507
msgid "_Deselect All"
msgstr "С_кінуць усё"

#: ../UpdateManager/UpdateManager.py:513
msgid "Select _All"
msgstr "Вылучыць _усе"

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr "%s будзе загружана."

#: ../UpdateManager/UpdateManager.py:584
#, fuzzy
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] "Абнаўленне ўжо загружана, але яшчэ не ўсталявана"
msgstr[1] "Абнаўленні ўжо загружаны, але яшчэ не ўсталяваныя"
msgstr[2] "Абнаўленьні ўжо запампаваныя, але яшчэ не ўсталяваныя"

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr ""

#: ../UpdateManager/UpdateManager.py:598
msgid "Unknown download size."
msgstr "Загружаны памер невядомы."

#: ../UpdateManager/UpdateManager.py:624
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Невядома, калі інфармацыя аб пакеце была абноўлена апошнім разам. Калі "
"ласка, націсніце кнопку \"Праверыць\" для абнаўлення гэтай інфармацыі."

#: ../UpdateManager/UpdateManager.py:630
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Інфармацыя аб пакетах была абноўлена %(days_ago)s дзён таму.\n"
"Націсніце кнопку «Праверыць» для праверкі новых абнаўленняў праграм."

#: ../UpdateManager/UpdateManager.py:635
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural "The package information was last updated %(days_ago)s days ago."
msgstr[0] "Інфармацыя аб пакетах была абноўлена %(days_ago)s дзень таму."
msgstr[1] "Інфармацыя аб пакетах была абноўлена %(days_ago)s дні таму."
msgstr[2] "Інфармацыя аб пакетах была абноўлена %(days_ago)s дзён таму."

#: ../UpdateManager/UpdateManager.py:639
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] "Інфармацыя аб пакетах была абноўлена %(hours_ago)s гадзіну таму."
msgstr[1] "Інфармацыя аб пакетах была абноўлена %(hours_ago)s гадзіны таму."
msgstr[2] "Інфармацыя аб пакетах была абноўлена %(hours_ago)s гадзінаў таму."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:644 ../UpdateManager/UpdateManager.py:646
#: ../UpdateManager/UpdateManager.py:648
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Інфармацыя аб пакеце была абноўлена прыкладна %s хвілін таму."

#: ../UpdateManager/UpdateManager.py:650
msgid "The package information was just updated."
msgstr "Інфармацыя аб пакеце была толькі што абноўлена."

#: ../UpdateManager/UpdateManager.py:689
msgid "Software updates may be available for your computer."
msgstr "Магчыма прысутнічаюць абнаўленні праграм для вашага кампутара."

#: ../UpdateManager/UpdateManager.py:697
#, python-format
msgid ""
"Updated software has been issued since %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:700
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:758
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Для абнаўлення патрабуецца %s вольнага месца на дыске «%s». Калі ласка, "
"вызваліце як мінімум %s дыскавай прасторы на «%s». Ачысціце вашу сметніцу і "
"выдаліце часовыя пакеты ўсталёвак, выканаўшы ў тэрмінале каманду 'sudo apt-"
"get clean'."

#: ../UpdateManager/UpdateManager.py:783
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Для завяршэння ўсталёўкі абнаўленняў неабходна перазапусціць кампутар. Калі "
"ласка, захавайце вынікі вашай працы перад тым, як працягнуць."

#: ../UpdateManager/UpdateManager.py:847
msgid "Reading package information"
msgstr "Чытанне інфармацыі аб пакетах"

#: ../UpdateManager/UpdateManager.py:862
msgid "Connecting..."
msgstr "Злучэнне..."

#: ../UpdateManager/UpdateManager.py:879
msgid "You may not be able to check for updates or download new updates."
msgstr "У вас недастаткова прывілеяў для праверкі ці зарузкі абнаўленняў."

#: ../UpdateManager/UpdateManager.py:1002
msgid "Could not initialize the package information"
msgstr "Не атрымалася атрымаць інфармацыю аб пакунках"

#: ../UpdateManager/UpdateManager.py:1003
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Падчас працы з пакетам паўстала невырашальная памылка.\n"
"\n"
"Калі ласка, паведаміце аб гэтай памылцы пакета «update-manager» і ўключыце "
"гэтае паведамленне:\n"

#: ../UpdateManager/UpdateManager.py:1032
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Крытычная памылка падчас апрацоўкі пакунка абнаўленняў.\n"
"\n"
"Калі ласка, паведаміце аб гэтай памылцы пакета 'Мэнэджар абнаўленняў' і "
"ўключыце гэтае паведамленне:"

#: ../UpdateManager/UpdateManager.py:1056
msgid " (New install)"
msgstr " (Новая ўсталёўка)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1063
#, python-format
msgid "(Size: %s)"
msgstr "(Памер: %s)"

#: ../UpdateManager/UpdateManager.py:1067
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "З версіі %(old_version)s на %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1071
#, python-format
msgid "Version %s"
msgstr "Версія %s"

#: ../UpdateManager/UpdateManager.py:1104 ../do-release-upgrade:112
msgid "Release upgrade not possible right now"
msgstr "Абнаўленьне выпуску зараз немагчымае"

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:113
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Абнаўленне рэлізу цяпер не можа быць выканана, паспрабуйце пазней. Сервер "
"паведаміў: '%s'"

#: ../UpdateManager/UpdateManager.py:1107 ../check-new-release-gtk:126
msgid "Downloading the release upgrade tool"
msgstr "Спампоўваецца праграма для абнаўлення дыстрыбутыва"

#: ../UpdateManager/UpdateManager.py:1114
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Даступны новы выпуск Ubuntu '%s'</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1153
msgid "Software index is broken"
msgstr "Пашкоджаны індэкс праграм"

#: ../UpdateManager/UpdateManager.py:1154
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Немагчыма ўсталяваць ці выдаліць праграмы. Калі ласка, выпраўце гэтую "
"праблему з дапамогай \"Synaptic\" альбо выканаўшы ў тэрмінале \"sudo apt-get "
"install -f\"."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Праверыць наяўнасць абнаўленняў"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Усталяваць ўсе даступныя абнаўленні"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Cancel"
msgstr "Скасаваць"

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Changelog"
msgstr "Гісторыя зменаў"

#: ../UpdateManagerText/UpdateManagerText.py:40
msgid "Updates"
msgstr "Абнаўленні"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid "Building Updates List"
msgstr "Фармуецца спіс абнаўленняў"

#: ../UpdateManagerText/UpdateManagerText.py:56
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Звычайнае абнаўленне ня можа быць выканана, калі ласка, запусціце: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
" Гэта можа быць выклікана:\n"
" * Папярэднім абнаўленнем, якое ня было завершана\n"
" * Праблемамі з асобна ўсталяванымі пакункамі альбо праграмамі\n"
" * Неафіцыйнымі пакункамі, якія не падтрымліваюцца Ubuntu\n"
" * Натуральнымі зменамі ў папярэдняй версіі Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:125
msgid "Downloading changelog"
msgstr "Загрузка спісу змяненняў"

#: ../UpdateManager/Core/MyCache.py:147
#, python-format
msgid "Other updates (%s)"
msgstr "Іншыя абнаўленні (%s)"

#: ../UpdateManager/Core/MyCache.py:325
msgid "This update does not come from a source that supports changelogs."
msgstr ""
"Гэтае абнаўленне пастаўляецца крыніцай, што не падтрымлівае спісы зменаў."

#: ../UpdateManager/Core/MyCache.py:331 ../UpdateManager/Core/MyCache.py:359
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Памылка пры загрузцы спісу змен. \n"
"Калі ласка, спраўдзьце вашае злучэнне з Інтэрнэтам."

#: ../UpdateManager/Core/MyCache.py:338
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Змяненні версіі:\n"
"Усталяваная версія: %s\n"
"Даступная версія: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:348
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Сьпіс зменаў ня ўтрымлівае звязаных з пакункам запісаў.\n"
"\n"
"Калі ласка, выкарыстоўвайце http://launchpad.net/ubuntu/+source/%s/%s/"
"+changelog\n"
"пакуль змены ня будуць даступны альбо паспрабуйце яшчэ раз пазней."

#: ../UpdateManager/Core/MyCache.py:353
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Спіс зменаў пакуль што недаступны.\n"
"\n"
"Калі ласка, выкарыстоўвайце http://launchpad.net/ubuntu/+source/%s/%s/"
"+changelog\n"
"пакуль змены ня будуць даступны альбо паспрабуйце яшчэ раз пазней."

#: ../UpdateManager/Core/UpdateList.py:51
msgid "Failed to detect distribution"
msgstr "Не атрымалася вызначыць дыстрыбутыў"

#: ../UpdateManager/Core/UpdateList.py:52
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Падчас праверкі вашае сістэмы адбылася памылка '%s'"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Important security updates"
msgstr "Важныя абнаўленні звязаныя з бяспекай"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Recommended updates"
msgstr "Пажаданыя абнаўленні"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Proposed updates"
msgstr "Прапанаваныя абнаўленні"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Backports"
msgstr "Адаптацыі з больш пазнейшых версій (backports)"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Distribution updates"
msgstr "Абнаўленні дыстрыбутыва"

#: ../UpdateManager/Core/UpdateList.py:72
msgid "Other updates"
msgstr "Іншыя абнаўленні"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
#, fuzzy
msgid "<big><b>Starting Software Updater</b></big>"
msgstr "<big><b>Стартуе Мэнэджар абнаўленняў</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Абнаўленні праграм выпраўляюць памылкі, уразлівасці і дадаюць новыя "
"магчымасці."

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "_Частковае абнаўленне"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Ня ўсе абнаўленні можна ўсталяваць</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Запусціць частковае абнаўленне дыстрыбутыва з усталёўкай па магчымасьці "
"максімальнай колькасці абнаўленняў.\n"
" Магчымыя прычыны:\n"
"\n"
" * Папярэдняя спроба абнаўлення дыстрыбутыва не завяршылася\n"
" * Праблема з якой-небудзь усталяванай праграмай\n"
" * Неафіцыйныя пакункі, не прадстаўляемыя Ubuntu\n"
" * Нармальныя змены перадрэлізнай версыі Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "Праве_рыць"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Вы павінны праверыць наяўнасць абнаўленняў  уручную</big></b>\n"
"\n"
"Ваша сістэма не правярае наяўнасць абнаўленняў аўтаматычна. Вы можаце "
"наладзіць гэта ў <i>Крыніцы ўсталёўкі</i> на ўкладцы <i>Абнаўленні</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "_Хаваць гэтую інфармацыю ў будучыні"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "Пра_цягваць"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Праца ад батарэі</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Ваш кампутар працуе ад батарэі. Вы ўпэўнены, што хочаце працягнуць?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Абнавіць"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Паказваць прагрэс для асобных файлаў"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
#, fuzzy
msgid "Software Updater"
msgstr "Абнаўленні праграмнага забеспячэння"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
#, fuzzy
msgid "Starting Software Updater"
msgstr "Абнаўленні праграмнага забеспячэння"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "Аб_навіць"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "абнаўленні"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Changes"
msgstr "Змены"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "Description"
msgstr "Апісанне"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
#, fuzzy
msgid "Details of updates"
msgstr "Апісанне абнаўлення"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Вы знаходзіцеся ў роўмінгу і з вас можа спаганяцца плата за дадзеныя, якія "
"перадаюцца пры абнаўленні."

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Бяспечней будзе падлучыць кампутар да крыніцы сілкавання перад пачаткам "
"абнаўлення."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Settings..."
msgstr "_Налады..."

#: ../data/gtkbuilder/UpdateManager.ui.h:35
#, fuzzy
msgid "_Install Now"
msgstr "Усталяваць"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Даступная новая версія Ubuntu. Хочаце абнавіць?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Не абнаўляць"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Спытаць пазней"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Так, Абнавіць зараз"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Вы адмовіліся ад абнаўлення да новай версіі Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
#, fuzzy
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Вы можаце абнавіць пазней, адчыніўшы Кіраўнік Абнаўленьняў і націснуўшы на "
"\"Абнавіць\"."

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Абнаўленні праграмнага забеспячэння"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Паказаць і ўсталяваць даступныя абнаўленні"

#: ../update-manager:66 ../update-manager-text:55 ../do-release-upgrade:51
msgid "Show version and exit"
msgstr "Паказаць версію і выйсці"

#: ../update-manager:69
msgid "Directory that contains the data files"
msgstr "Тэчка, якая ўтрымлівае файлы дадзеных"

#: ../update-manager:72
msgid "Check if a new Ubuntu release is available"
msgstr "Праверыць, ці даступны новы рэліз Ubuntu"

#: ../update-manager:75 ../do-release-upgrade:54 ../check-new-release-gtk:186
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Праверка магчымасці абнаўлення да апошняй нестабільнай версіі дыстрыбутыва"

#: ../update-manager:79
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Абнаўленне з выкарыстаннем апошняй версіі Мэнэджара абнаўленняў"

#: ../update-manager:86
msgid "Do not focus on map when starting"
msgstr "Не выбіраць картку пры запуску"

#: ../update-manager:89
msgid "Try to run a dist-upgrade"
msgstr "Паспрабаваць запусціць dist-upgrade"

#: ../update-manager:92
msgid "Do not check for updates when starting"
msgstr "Не правяраць абнаўленні пад час запуску"

#: ../update-manager:96 ../do-release-upgrade:70
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Пратэставаць абнаўленне ў бяспечным рэжыме"

#: ../update-manager:116
msgid "Running partial upgrade"
msgstr "Ідзе частковае абнаўленне"

#: ../update-manager-text:59
msgid "Show description of the package instead of the changelog"
msgstr ""

#: ../do-release-upgrade:58 ../check-new-release-gtk:190
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Паспрабуйце абнавіцца да самага апошняга выпуска з дапамогай $distro-proposed"

#: ../do-release-upgrade:62
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Запусьціць у спэцыяльным рэжыме абнаўлення.\n"
"Зараз падтрымліваецца рэгулярнае абнаўленьне для «настольных» і «серверных» "
"сістэм."

#: ../do-release-upgrade:68
msgid "Run the specified frontend"
msgstr "Запусціць пазначаны інтэрфейс"

#: ../do-release-upgrade:73
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Правяраць наяўнасць новай версіі дыстрыбутыва і вярнуць вынік з дапамогай "
"кода выхаду"

#: ../do-release-upgrade:87
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:101
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Каб атрымаць інфармацыю аб абнаўленні, наведайце:\n"
"%(url)s\n"

#: ../do-release-upgrade:107
msgid "No new release found"
msgstr "Новы выпуск ня знойдзены"

#: ../do-release-upgrade:119
#, c-format
msgid "New release '%s' available."
msgstr "Даступны новы выпуск '%s'."

#: ../do-release-upgrade:120
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Выканайце «do-release-upgrade», каб абнавіцца да яго."

#: ../check-new-release-gtk:101
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Даступнае абнаўленне Ubuntu %(version)s"

#: ../check-new-release-gtk:143
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Вы адмовіліся ад абнаўлення да Ubuntu %s"

#: ../check-new-release-gtk:196
msgid "Add debug output"
msgstr ""

#: ../ubuntu-support-status:91
msgid "Show unsupported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:94
msgid "Show supported packages on this machine"
msgstr ""

#: ../ubuntu-support-status:97
msgid "Show all packages with their status"
msgstr ""

#: ../ubuntu-support-status:100
msgid "Show all packages in a list"
msgstr ""

#: ../ubuntu-support-status:142
#, c-format
msgid "Support status summary of '%s':"
msgstr ""

#: ../ubuntu-support-status:145
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr ""

#: ../ubuntu-support-status:151
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr ""

#: ../ubuntu-support-status:154
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr ""

#: ../ubuntu-support-status:162
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""

#: ../ubuntu-support-status:166
msgid "No longer downloadable:"
msgstr ""

#: ../ubuntu-support-status:169
msgid "Unsupported: "
msgstr ""

#: ../ubuntu-support-status:174
#, c-format
msgid "Supported until %s:"
msgstr "Падтрымліваюцца да %s:"

#: ../ubuntu-support-status:183
msgid "Unsupported"
msgstr ""

#. Why do we use %s here instead of $strings or {} format placeholders?
#. It's because we don't want to break existing translations.
#: ../janitor/plugincore/exceptions.py:42
#, python-format
msgid "Unimplemented method: %s"
msgstr "Нявыкананы метад: %s"

#: ../janitor/plugincore/core/file_cruft.py:41
msgid "A file on disk"
msgstr "Файл на дыску"

#: ../janitor/plugincore/core/missing_package_cruft.py:39
msgid "Install missing package."
msgstr "Усталяваць адсутнічаючыя пакункі."

#. 2012-06-08 BAW: i18n string; don't use {} or PEP 292.
#: ../janitor/plugincore/core/missing_package_cruft.py:49
#, python-format
msgid "Package %s should be installed."
msgstr "Неабходна усталяваць пакунак %s."

#: ../janitor/plugincore/core/package_cruft.py:49
msgid ".deb package"
msgstr ".deb пакет"

#: ../janitor/plugincore/plugins/langpack_manual_plugin.py:46
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s павінен быць пазначаны як усталяваны ўручную."

#: ../janitor/plugincore/plugins/kdelibs4to5_plugin.py:49
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Пры абнаўленні, калі ўсталяваны kdelibs4-dev, то неабходна ўсталяваць "
"kdelibs5-dev. Гэта баґ, аб якім можна пачытаць тут — bugs.launchpad.net, bug "
"#279621"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:44
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "Састарэлы змест %i у файле статусу"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:47
msgid "Obsolete entries in dpkg status"
msgstr "Састарэлы змест у статусе dpkg"

#. pragma: no cover
#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:50
msgid "Obsolete dpkg status entries"
msgstr "Састарэлы змест станаў dpkg"

#: ../janitor/plugincore/plugins/remove_lilo_plugin.py:42
msgid "Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Выдаліце lilo, калі grub ужо ўсталяваны. (Глядзіце bug #314004 для дэталяў)"

#~ msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
#~ msgstr "<b><big>Абнаўленне Ubuntu да версіі 12.04</big></b>"

#~ msgid "%(count)s update has been selected."
#~ msgid_plural "%(count)s updates have been selected."
#~ msgstr[0] "%(count)s абнаўленне абрана."
#~ msgstr[1] "%(count)s абнаўленняў абрана."
#~ msgstr[2] "%(count)s абнаўленняў абрана."

#~ msgid "%(count_str)s %(download_str)s"
#~ msgstr "%(count_str)s %(download_str)s"

#~ msgid "Welcome to Ubuntu"
#~ msgstr "Вітаем у Ubuntu"

#~ msgid "Update Manager"
#~ msgstr "Мэнэджар абнаўленняў"

#~ msgid "Starting Update Manager"
#~ msgstr "Запуск Мэнэджара абнаўленняў"

#~ msgid "You are connected via a wireless modem."
#~ msgstr "Вы падлучаны праз бесправадны мадэм."

#~ msgid "_Install Updates"
#~ msgstr "_Усталяваць абнаўленні"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Праверыць наяўнасьць новага выпуску ubuntu"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Абнаўленне адбываецца ў бяспечным (тэставым) рэжыме. Усе змены "
#~ "запісваюцца ў '%s' і будуць сцёртыя пасля перазагрузкі.\n"
#~ "\n"
#~ "*Ніякія* змены сістэмных каталогаў ад гэтага часу да наступнай "
#~ "перазагрузкі не зяўляюцца пастаяннымі."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal."
#~ msgstr ""
#~ "Абнаўленне звестак аб сховішчы прывяло да стварэння няспраўнага файла. "
#~ "Калі ласка, адпраўце гэту справаздачу, выкарыстоўваючы ў тэрмінале "
#~ "каманду 'ubuntu-bug update-manager'"

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Калі ласка, адпраўце гэтае паведамленне, выкарыстоўваючы ў тэрмінале "
#~ "каманду 'ubuntu-bug update-manager' і дадайце файлы з /var/log/dist-"
#~ "upgrade/ у паведамленне аб памылцы.\n"
#~ "%s"

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The "
#~ "upgrade will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug using the command 'ubuntu-bug update-manager' "
#~ "in a terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Сістэма не змагла загрузіць спіс абавязковых патрабаванняў для выканання "
#~ "абнаўлення. Праграма абнаўлення скончыць сваю працу і верне сістэму да яе "
#~ "зыходнага стану.\n"
#~ "\n"
#~ "Калі ласка, паведаміце аб памылцы, выкарыстоўваючы ў тэрмінале каманду "
#~ "'ubuntu-bug update-manager' і дадайце файлы з /var/log/dist-upgrade/ у "
#~ "справаздачу аб сапсаванасці."

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this using the "
#~ "command 'ubuntu-bug update-manager' in a terminal and include the files "
#~ "in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Немагчыма падрыхтаваць сістэму да выканання абнаўлення. Калі ласка, "
#~ "паведаміце аб памылцы, выкарыстоўваючы ў тэрмінале каманду 'ubuntu-bug "
#~ "update-manager' і дадайце файлы з /var/log/dist-upgrade/ у справаздачу аб "
#~ "сапсаванасці."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug using the command "
#~ "'ubuntu-bug update-manager' in a terminal and include the files in /var/"
#~ "log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Пасля абнаўлення звестак аб пакеце, асноўны пакет '%s' больш не можа быць "
#~ "знойдзены.\n"
#~ "Гэта азначае сур'ёзную памылку; калі ласка адпраўце справаздачу аб гэтай "
#~ "праблеме, выкарыстоўваючы каманду 'ubuntu-bug update-manager' у тэрмінале "
#~ "і далучыце файлы з /var/log/dist-upgrade да паведамлення аб памылцы."

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
#~ msgstr ""
#~ "Ваша графічнае абсталяванне можа не цалкам падтрымлівацца ў Ubuntu 11/04."

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Загрузка і ўсталёўка абнаўленняў можа заняць некалькі гадзін. Пасля "
#~ "завяршэння загрузкі такі працэс не можа быць адменены."

#~ msgid "Your system is up-to-date"
#~ msgstr "Ваша сістэма не патрабуе абнаўлення"

#~ msgid ""
#~ "You will not get any further security fixes or critical updates. Please "
#~ "upgrade to a later version of Ubuntu Linux."
#~ msgstr ""
#~ "У далейшым вы не будзеце атрымліваць абнаўленні бяспекі ці крытычныя "
#~ "абнаўленні. Калі ласка, абнавіцеся да апошняй версіі Ubuntu."

#~ msgid "There are no updates to install"
#~ msgstr "Няма абнаўленняў"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from Applications later."
#~ msgstr ""
#~ "Гэтыя абнаўленні былі выпушчаныя пасля дадзенай версіі Ubuntu. Калі вы не "
#~ "маеце жадання ўсталёўваць іх цяпер, вы можаце пазней абраць \"Мэнэджар "
#~ "абнаўленняў\" з меню Праграмы."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from the Administration Menu later."
#~ msgstr ""
#~ "Гэтыя абнаўленні былі выпушчаныя пасля дадзенай версіі Ubuntu. Калі вы не "
#~ "маеце жадання ўсталёўваць іх цяпер, вы можаце пазней абраць \"Мэнэджар "
#~ "абнаўленняў\" з меню Адміністраванне."

#~ msgid "Software updates are available for this computer"
#~ msgstr ""
#~ "У наяўнасці маюцца абнаўленні праграмнага забеспячэння для гэтага "
#~ "кампутара"

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Калі вы не маеце жадання ўсталёўваць іх цяпер, вы можаце пазней абраць "
#~ "\"Мэнэджар абнаўленняў\" з меню Сістэма → Адміністраванне."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from "
#~ "Applications later."
#~ msgstr ""
#~ "Калі вы не маеце жадання ўсталёўваць іх цяпер, абярыце \"Мэнэджар "
#~ "абнаўленняў\" з меню «Прыкладанні» пазней."

#~ msgid "0 kB"
#~ msgstr "0 kB"

#~ msgid "1 kB"
#~ msgstr "1 kB"

#~ msgid ""
#~ "The support in Ubuntu 11.04 for your intel graphics hardware is limited "
#~ "and you may encounter problems after the upgrade. Do you want to continue "
#~ "with the upgrade?"
#~ msgstr ""
#~ "Падтрымка інтэграванага графічнага абсталявання Intel у Ubuntu 11.04 можа "
#~ "быць абмежаванай, а таксама можа прывесці да паўстання памылак. Вы "
#~ "сапраўды хочаце працягнуць абнаўленне?"

#~ msgid "%.0f kB"
#~ msgstr "%.0f kB"
